.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Feature Request
+about: Suggest an idea for this project	about
+title: ''
+labels: kind/feature-request
+assignees: ''
+
+---
+
```

Comment:
`A clear and concise` too wordy is not needed, and maybe ironic?

Reply:
+1 to making this more concise. I'd suggest the following should be sufficient:

```markdown
**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** Which persona is this feature for?

**Problem**
A short explanation of the problem, including relevant restrictions.

**Exit Criteria**
A measurable (binary) test that would indicate that the problem has been resolved.

**Time Estimate (optional):** How many developer-days do you think this may take to resolve?
```

I'd prefer to have the different solutions be followups to the initial issue, rather than needing a solution decided in order to propose a missing feature.

Reply:
Most of the wordings here were all auto-generated by Github as these are standard templates. I would prefer to keep it that way rather. It helps folks creating an issue to think thoroughly which will help in planning. However I would change certain typos and trim down the text for fields that I added over what Github provided